apps/camera/style/recording-timer.css

```diff
@@ -5,6 +5,7 @@
   left: 0;
   z-index: 1;
 
+  visibility: hidden;
```

Comment:
I think it actually works both ways, notice how transition delays are handled in both cases. Yours is slightly different as your visibility is inversed, so you would need to place/adjust the delays accordingly.